client/src/app/employee/employee-list/employee-list.component.ts

```diff
@@ -13,21 +22,21 @@ import { EmployeeListService } from './state/employee-list.service';
 export class EmployeeListComponent implements OnInit, OnDestroy {
   readonly faSkull = faSkull;
   readonly faSmileBeam = faSmileBeam;
-  readonly employees: Observable<Employee[]>;
+  readonly faPlus = faPlus;
 
-  constructor(
-    private service: EmployeeListService,
-    private query: EmployeeListQuery,
-  ) {
-    this.employees = this.query.selectAll();
-    this.service.getEmployees();
-  }
+  @Input()
+  employees: Observable<EmployeeList[]>;
```

Comment:
You probably want `@Input() employees: EmployeeList[]` here, not an `Observable<>`

Reply:
fixed!